Meeting on 2024-10-22 (#161)
OpenRailAssociation/technical-committee: meetings/2024-10-22.md

# TC Meeting 2024-10-22

Present: Florian, Cornelius, Peter; Excused: Max, Frederik, Loic, Tristram, Stephanie, Adrian

## Agenda

* Onboarding (https://github.com/OpenRailAssociation/technical-committee/labels/Onboarding)
* Questionnaire for incubation stage 2 (https://github.com/OpenRailAssociation/technical-committee/pull/159)

## Protocol

* Onboarding status see issues
* Questionnaire for stage 2

>   * We agree that we change the questionnaire to a form where it's mostly a checklist, where the project submits metrics or links to material which shows that the criteria are fullfilled. Plus the opportunity to add additional comments, e.g. for explaining or interpreting metrics.

Reviewer: ```suggestion
  * We agree that we change the questionnaire to a form where it is mostly a checklist, where the project submits metrics or links to material which shows that the criteria are fullfilled. Plus the opportunity to add additional comments, e.g., for explaining or interpreting metrics.
```

  * The project submits the information for the checklist and the TC goes through the list and checks that criteria are met
  * Cornelius will adapt the questionnaire in the pull request

>   * We discusssed all the open questions in the pull request and have a path to move forward now.

Reviewer: ```suggestion
  * We discussed all the open questions in the pull request and have a path to move forward now.
```

>   * Cornelius will adress the reamining TODOs from the pull request and then merge it. All the remaining things we need to discuss for incubation stage 2 are tracked in the milestone https://github.com/OpenRailAssociation/technical-committee/milestone/5

Reviewer: ```suggestion
  * Cornelius will adress the remaining TODOs from the pull request and then merge it. All the remaining things we need to discuss for incubation stage 2 are tracked in the milestone https://github.com/OpenRailAssociation/technical-committee/milestone/5
```
